site/styles/_variables.scss

```diff
@@ -50,6 +50,12 @@
   --page-detail-padding-top: 120px;
   --page-detail-padding-top-md: 120px;
   --page-detail-padding-top-lg: 180px;
+
+  --font-size-body-1-style: 16px;
```

Comment:
If we are going to have several exposed variables per mixin, it might be easier to read if you have the property name after the mixin name e.g. `--body-1-style-font-size: 16px;`

Reply:
Also, it's probably worth exposing the sizes for all of the mixins (you didn't do the headings or overline mixins).

Reply:
I think `--font-size-body-1-style` looks more readable than `--body-1-style-font-size`

Reply:
This is much better:
```
--body-1-style-letter-spacing: normal;
--body-1-style-font-size: 16px;
--body-2-style-letter-spacing: normal;
--body-2-style-font-size: 24px;
```

Than this:
```
--letter-spacing-body-1-style: normal;
--font-size-body-1-style: 16px;
--letter-spacing-body-2-style: normal;
--font-size-body-2-style: 24px;
```

Reply:
@PeculiarGoat The headings one has got different font sizes for various breakpoints. I think it's going to just complicate if we end up ending 10 vars for just the heading

Reply:
It's going to get complicated if we expose some font sizes but not others... they should all be exposed.

Reply:
But some of the headings are never going to get changed like the `small` in `heading-1`. Are you suggesting of adding a var for `heading-1-style`, `heading-1-style-desktop`, `heading-1-style-small`, etc?

Reply:
Yes please.  But they should have the breakpoint labels rather than `desktop`, `tablet`, etc.